### PR TITLE
WINDUP-822: Overview page should list projects with collapsable div

### PR DESCRIPTION
**Pull request by mbriskar**

### CODE REVIEW

**Comment by mbriskar** [comment]
screenshot from 2015-11-10 12-53-47

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/580/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1208/

File: test-util/src/main/java/org/jboss/windup/testutil/html/TestJavaApplicationOverviewUtil.java [hunk 1]
```diff
@@ -179,8 +184,35 @@ public void checkFilePathAndTag(String appSection, String filePath, String tag)
...
+    }
+
+    /**
+     * Returns the xpath full path of the given element. E.g something like /html/body/div[2]/p
+     * @param driver
+     * @param element
+     * @return
+     */
+    private String getElementXPath(WebDriver driver, WebElement element) {
+        return "/html/" + (String)((JavascriptExecutor)driver).executeScript("gPt=function(c){if(c.id!==''){return'id(\"'+c.id+'\")'}if(c===document.body){return c.tagName}var a=0;var e=c.parentNode.childNodes;for(var b=0;b<e.length;b++){var d=e[b];if(d===c){return gPt(c.parentNode)+'/'+c.tagName+'['+(a+1)+']'}if(d.nodeType===1&&d.tagName===c.tagName){a++}}};return gPt(arguments[0]).toLowerCase();", element);
```
Comment (jsight) [inline test-util/src/main/java/org/jboss/windup/testutil/html/TestJavaApplicationOverviewUtil.java]: Lol, figuring that out had to be painful.

Comment (mbriskar) [inline test-util/src/main/java/org/jboss/windup/testutil/html/TestJavaApplicationOverviewUtil.java]: most of it is copied from stackoverflow answer :)

**Pull request merged**